src/agent.cpp

```diff
@@ -45,6 +58,12 @@ namespace Rep
             {
                 return *this;
             }
+            // leading wildcard?
+            if (query.front() == '*')
+            {
+                Url::Url trimmed(trim_front(query, '*'));
+                directives_.push_back(Directive(escape_url(trimmed), false));
+            }
```

Comment:
Did you mean for there to be an `else` here? As this stands, you're pushing both the trimmed and untrimmed versions.

Reply:
Yes, that is the intent. There is an explanation in my PR comment.

Reply:
D'oh again!